<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImCalibrateSettingWidget</class>
 <widget class="QWidget" name="ImCalibrateSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1605</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="CalSetting_range_5">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>1091</width>
     <height>40</height>
    </rect>
   </property>
   <property name="title">
    <string>校准曲线管理</string>
   </property>
   <property name="borders" stdset="0">
    <string notr="true">group_tit</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>792</y>
     <width>1556</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1</width>
     <height>793</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_2">
   <property name="geometry">
    <rect>
     <x>1550</x>
     <y>0</y>
     <width>1</width>
     <height>793</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1556</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string>solid</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>5</y>
     <width>451</width>
     <height>781</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="CalSetting_change_over">
      <property name="title">
       <string>自动校准设置</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="borders" stdset="0">
       <string>group_tit</string>
      </property>
      <widget class="QCheckBox" name="change_over_bottle_ckbox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>换瓶校准</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="change_over_lot_ckbox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>160</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>换批校准</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>试剂更换</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string/>
      </property>
      <property name="borders" stdset="0">
       <string>all</string>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>校准曲线过期</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="lot_timeout_ckbox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>批曲线过期</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="bottl_timeout_ckbox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>瓶曲线过期</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <property name="borders" stdset="0">
       <string>all</string>
      </property>
      <widget class="QCheckBox" name="union_qc_cali_mode_ckbox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>联合质控失控</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>质控失控</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="single_qc_cali_mode_ckbox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>单质控失控</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidgetbatchcurev">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>47</y>
     <width>1091</width>
     <height>741</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string/>
        </property>
        <property name="borders" stdset="0">
         <string>all</string>
        </property>
        <widget class="QLineEdit" name="lot_valid_time_edit">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>90</y>
           <width>180</width>
           <height>40</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_22">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>50</y>
           <width>471</width>
           <height>40</height>
          </rect>
         </property>
         <property name="text">
          <string>成为批曲线时间限(小时)</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string/>
        </property>
        <property name="borders" stdset="0">
         <string>all</string>
        </property>
        <widget class="QCheckBox" name="cali_failed_ckbox">
         <property name="geometry">
          <rect>
           <x>45</x>
           <y>83</y>
           <width>391</width>
           <height>40</height>
          </rect>
         </property>
         <property name="text">
          <string>校准屏蔽</string>
         </property>
         <property name="checks" stdset="0">
          <string>push</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="CalSetting_range">
      <property name="title">
       <string>校准参数</string>
      </property>
      <property name="borders" stdset="0">
       <string>group_tit</string>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>751</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>校准品A线性校准Δ范围值（Δ1）</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>215</x>
         <y>190</y>
         <width>20</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>150</y>
         <width>721</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>校准品B线性校准Δ范围值（Δ2）</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>420</y>
         <width>731</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>校准品B测定值方差范围（CV2）</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>240</y>
         <width>741</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>校准品线性校准判定值范围K（=Δ1/Δ2）</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>215</x>
         <y>278</y>
         <width>20</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>330</y>
         <width>731</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>校准品A测定值方差范围（CV1）</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>215</x>
         <y>98</y>
         <width>20</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="CV2Edit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>460</y>
         <width>170</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lower_delta2_edit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>192</y>
         <width>170</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="CV1Edit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>370</y>
         <width>170</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lower_k_edit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>280</y>
         <width>170</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="upper_k_edit">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>280</y>
         <width>170</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lower_delta1_edit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>170</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="upper_delta2_edit">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>192</y>
         <width>170</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="upper_delta1_edit">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>100</y>
         <width>170</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lot_valid_time_edit</tabstop>
  <tabstop>cali_failed_ckbox</tabstop>
  <tabstop>lower_delta1_edit</tabstop>
  <tabstop>upper_delta1_edit</tabstop>
  <tabstop>lower_delta2_edit</tabstop>
  <tabstop>upper_delta2_edit</tabstop>
  <tabstop>lower_k_edit</tabstop>
  <tabstop>upper_k_edit</tabstop>
  <tabstop>CV1Edit</tabstop>
  <tabstop>CV2Edit</tabstop>
  <tabstop>change_over_bottle_ckbox</tabstop>
  <tabstop>change_over_lot_ckbox</tabstop>
  <tabstop>bottl_timeout_ckbox</tabstop>
  <tabstop>lot_timeout_ckbox</tabstop>
  <tabstop>single_qc_cali_mode_ckbox</tabstop>
  <tabstop>union_qc_cali_mode_ckbox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
